doc/rst/source/gmt.conf.rst

```diff
@@ -57,26 +57,26 @@ COLOR Parameters
 
     **COLOR_BACKGROUND**
         Color used for the background of images (i.e., when z < lowest color
-        table entry) [black].
+        table entry) [*default is <black>*].
```

Comment:
A single asterisk make it *italic*. I though the idea was to make it **bold**.
Also, I don't think we want to wrap it under ``<  >``

Reply:
I agree with you that the value should be bolded. Do you approve of all the defaults being formatted like "[default is **black**]" rather than just "[**black**]" @joa-quim ?

Reply:
Yes, I think it's better to be clear saying **this is the default**  than being **implicit**

Reply:
All the current changes have the "default is" part *italicized* (e.g., [*default is* **black**]). We need to make a decision about whether these should be italics or normal font. My preference is normal font (e.g., [default is **black**]) because italics do not show up in these strings elsewhere in the docs. Thoughts @joa-quim?

It should be an easy search and replace fix if these need to be corrected.